diff_cover/diff_reporter.py

```diff
@@ -72,20 +75,28 @@ def _is_path_excluded(self, path):
         """
         Check if a path is excluded.
 
+        First it is checked if the path matches one of the include patterns (if provided).
+        Second, the path is matched against the exclude patterns.
+
         :param str path:
-            Path to check against the exclude patterns.
+            Path to check against the exclude and include patterns.
         :returns:
-            True if there are exclude patterns and the path matches,
-            otherwise False.
+            True if the patch should be excluded, otherwise False.
         """
+        absolute_path = os.path.abspath(path)
+        include = self._include
+        if include:
+            for pattern in include:
+                if not any(path in glob.glob(pattern, recursive=True)):
+                    return True
+
```

Comment:
So far none of the tests area actually hitting this code path. Given this is the core of the feature at least one would be good

Reply:
done